README.md

```diff
@@ -18,5 +18,5 @@ This library should compile with any combination of features with Rust 1.63.0.
 To build with the MSRV you will need to pin dependencies as follows:
 
 ```shell
-cargo update -p home --precise 0.5.5
+cargo update -p jobserver --precise "0.1.26"
```

Comment:
@notmandatory I don't think I get it 🤔 
We only need to pin the `jobserver` but we are pinning some others on the CI ?

Reply:
We only need to pin jobserver to compile with msrv. The others are only needed to run the tests (in CI) with the msrv.